pipelinejobs.groovy

```diff
@@ -0,0 +1,37 @@
+// Import the utility functionality.
```

Comment:
nit: Since we are going to start having new groovy files can we put them in a sub-dir, perhaps under buildpipeline directory?